src/raven/client.clj

```diff
@@ -103,11 +103,10 @@
 (defn default-payload
   "Provide default values for a payload."
   [ts]
-  {:level       "off"
+  {:level       "error"
    :server_name (localhost)
-   :culprint    "<none>"
-   :platform    "java"
-   :timestamp   ts})
```

Comment:
If this goes, so should the argument IMO

Reply:
Yeah, either that, or fixing the timestamp format to make sentry understand it.